packages/content/types/query-builder.d.ts

```diff
@@ -7,6 +6,20 @@ interface QueryBuilderOptions {
   postprocess?: any[];
 }
 
+interface FetchReturn extends Record<string, any> {
+  dir: string;
+  path: string;
+  extension: '.md' | '.json' | '.yaml' | '.xml' | '.csv' | string;
+  slug: string;
+  createdAt: string;
+  updatedAt: string;
+  toc?: {
+    id: string;
+    depth: number;
+    text: string;
+  }[];
```

Comment:
This should probably be added to `IContentDocument` instead of duplicating it here.

Reply:
I will introduce `IContentDocumentBase` to avoid code duplication.
And add missing `body` field too.